src/components/utils/popover.ts

```diff
@@ -163,19 +170,31 @@ export default class Popover extends EventsDispatcher<PopoverEvent> {
    * @param options.className - additional class name to be added to the popover wrapper
    * @param options.filterLabel - label for the search Field
    * @param options.nothingFoundLabel - label of the 'nothing found' message
+   * @param options.customContent - arbitrary html element to be inserted before items list
+   * @param options.customContentFlippableItems - list of html elements inside custom content area that should be available for keyboard navigation
+   * @param options.scopeElement - editor container element
+   * @param shouldHighlightActivated - If true, activated item should be highlighted as active once clicked.
    */
-  constructor({ items, className, searchable, filterLabel, nothingFoundLabel }: {
+  constructor({ items, className, searchable, filterLabel, nothingFoundLabel, customContent, customContentFlippableItems, scopeElement, shouldHighlightActivated = false }: {
```

Comment:
Looks like the builder pattern might be useful here

Reply:
Is it big number of constructor params that bothers you? They are wrapped in object, so, client will only have to specify those props that are actually needed. What's the advantage of using builder pattern here?

Reply:
Builder pattern would be helpful in case of common use cases of this class. Moreover, Director pattern might be useful as well. Anyway I think refactoring is required so maybe final version would be ok without Builder/Director